src/libraries/System.DirectoryServices.Protocols/src/System/DirectoryServices/Protocols/ldap/LdapSessionOptions.cs

```diff
@@ -524,6 +524,7 @@ public VerifyServerCertificateCallback VerifyServerCertificate
             }
         }
 
+        // In practice, this apparently rarely if ever contains useful text
```

Comment:
Not that you need to fix it here, but LdapSessionOptions currently has a split implementation between Linux and Windows, so if the server error is retrieved with different options, we can move the implementation of this internal string to the RID-specific files and use the right option on each case.